src/csharp/Microsoft.Spark/Sql/Column.cs

```diff
@@ -743,6 +743,167 @@ public Column Over()
             return Apply("over");
         }
 
+        /// <summary>
+        ///  A boolean expression that is evaluated to true if the value of this expression 
+        ///  is contained by the evaluated values of the arguments.  
+        /// </summary>
+        /// <param name="list">List of values to check the column against</param>
+        /// <returns>Column object</returns>
+        public Column IsIn(params string[] list){
```

Comment:
Please follow the existing coding style.

Reply:
newline now before braces - sorry didn't see it for the life of me in the original functions!